pwa/app/routes/apidocs.tsx

```diff
@@ -0,0 +1,308 @@
+import { Link } from '@remix-run/react';
+
+export default function ApiDocs(): JSX.Element {
+  return (
+    <>
+      <div className="flex flex-col divide-y [&_*]:mt-4">
+        <div>
+          <h1 className="text-3xl font-medium">
+            The Blue Alliance Developer APIs
+          </h1>
+          <p>
+            The Blue Alliance cares about making our data publicly accessible
+            via our various APIs. We want to help inspire people to build their
+            own projects and get started with data analysis and software
+            development. This page explains the APIs we provide and how to get
+            started using them.
+          </p>
+        </div>
+
+        <div>
+          <h3 className="text-2xl">Need Help?</h3>
+          <p>
```

Comment:
Is this section intentionally different from what's on the current website?

Reply:
Yes. I recall you mentioning some of the contact links possibly being broken (the slack, google group) in another pr and i think it makes more sense to link the contact page with the same resources instead of relinking them. I also made the information in the other bullet points more concise.